src/goto-programs/string_instrumentation.h

```diff
@@ -41,16 +41,19 @@ class incorrect_source_program_exceptiont : public cprover_exception_baset
 void string_instrumentation(
   symbol_tablet &,
   message_handlert &,
-  goto_programt &);
+  goto_programt &,
+  size_t);
```

Comment:
:pick: naming variables in the header (particularly ones where the type tells you nothing) helps users of the function to figure out what they should be passing in